data.sql

```diff
@@ -3,4 +3,12 @@
 INSERT INTO ANIMALS (ID,NAME,DATE_OF_BIRTH,ESCAPE_ATTEMPTS,NEUTERED,WEIGHT_KG) VALUES (1, 'Agumon', '2020-02-03',  0, true, 10.23);
 INSERT INTO ANIMALS (ID,NAME,DATE_OF_BIRTH,ESCAPE_ATTEMPTS,NEUTERED,WEIGHT_KG) VALUES (2, 'Gabumon', '2018-11-15',  2, true, 8);
 INSERT INTO ANIMALS (ID,NAME,DATE_OF_BIRTH,ESCAPE_ATTEMPTS,NEUTERED,WEIGHT_KG) VALUES (3, 'Pikachu', '2021-01-07',  1, false, 15.04);
-INSERT INTO ANIMALS (ID,NAME,DATE_OF_BIRTH,ESCAPE_ATTEMPTS,NEUTERED,WEIGHT_KG) VALUES (4, 'Devimon', '2017-05-12',  5, true, 11);
\ No newline at end of file
+INSERT INTO ANIMALS (ID,NAME,DATE_OF_BIRTH,ESCAPE_ATTEMPTS,NEUTERED,WEIGHT_KG) VALUES (4, 'Devimon', '2017-05-12',  5, true, 11);
+
+INSERT INTO ANIMALS (ID,NAME,DATE_OF_BIRTH,ESCAPE_ATTEMPTS,NEUTERED,WEIGHT_KG) VALUES (5, 'Charmander', '2020-02-08',  0, false, -11);
+INSERT INTO ANIMALS (ID,NAME,DATE_OF_BIRTH,ESCAPE_ATTEMPTS,NEUTERED,WEIGHT_KG) VALUES (6, 'Plantmon', '2021-11-15',  2, true, -5.7);
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Good job making changes required by the previous reviewer. But as he mentioned to add the right value on the `WEIGHT_KG` column, you should make sure you update every where(i.e. both in your database and in the expected file). Once you added the required value in your database, you should even update your screenshots so we will make sure you correctly did it. I really suggest you update your screenshots from your PR before merging it as the previous reviewer didn't mentioned this detail in the previous reviewer.